pkg/network/tracer/tracer_linux_test.go

```diff
@@ -2068,3 +2069,67 @@ func TestBlockingReadCounts(t *testing.T) {
 
 	assert.Equal(t, uint64(n), conn.MonotonicSum().RecvBytes)
 }
+
+// GoTLS test
+
+func TestHTTPGoTLSCaptureNewProcess(t *testing.T) {
+	const ClientBin string = "./testutil/cmd/gotls_client/gotls_client"
+	const ServerAddr string = "localhost:8081"
+	const ExpectedOccurrences int = 10
```

Comment:
why are those consts exported (starts with capital letter)?

Reply:
Those are not exported actually. [Here](https://go.dev/play/p/WclEL9IM6lc) is a small example demonstrating this. This is also how they are demonstrated in the Go tour.
If you prefer those to start with a lowercase letter that is fine by me, though I would argue this might help differentiate constants from regular variables when reading the code. What do you think?